guides/AE2CableManagement.md

```diff
@@ -0,0 +1,71 @@
+Author: Shadoyama#4166
+
+# The Basics
+
+In this section, I'll be going over the basics of cabling and channels, just to make sure everyone understands or is on the same page as I am. If you understand the different types of cables, channels, and other basic stuff, this can be skipped.
+
+## Types of Cable
+
+This section will talk about the different types of cable and the advantages and disadvantages of using each type.
+
+Firstly, there are 4 different cables from Applied Energistics, and 2 from Ender IO. Here are the following from AE2:
+- Quartz Fiber
+- ME Glass Cable
+- ME Smart Cable
+- ME Dense Smart Cable
+
+EnderIO has both the ME Smart Cable and ME Dense Smart Cable, but they are both conduits, allowing for energy conduits and any other conduit from EnderIO to run alongside it in the same block. However, as I'll note in the next section, they have differences.
```

Comment:
Enderio conduits are more like the covered variants since they do not show channels.

Reply:
ME cable and ME Dense cable